lib/pure/terminal.nim

```diff
@@ -462,15 +509,35 @@ elif not defined(windows):
     result = stdin.readChar()
     discard fd.tcsetattr(TCSADRAIN, addr oldMode)
 
+# Deprecated procs assuming output to stdout:
+template setCursorPos*(x, y: int) {.deprecated.} = setCursorPos(stdout, x, y)
```

Comment:
I prefer an optional 'deprecated' then, like I did it for '-d:nimTableGet' so people can update their code if they really want to. Believe it or not, not everybody cares about 'stderr'. ;-)